src/tools/rust-installer/.gitignore

```diff
@@ -0,0 +1,5 @@
+*~
+tmp
+target/
+**/*.rs.bk
+Cargo.lock
```

Comment:
Add `rust-installer` to https://github.com/rust-lang/rust/blob/master/Cargo.toml or keep `Cargo.lock`?

Reply:
It's already a member of the top-level workspace. We could adjust the ignore file here to remove these but I don't think it's worth worrying over.